src/utils.js

```diff
@@ -0,0 +1,9 @@
+export let addZero = num => {
+  return num < 10 ? `0${num}` : num
+}
+
+export let getFormate = seconds => {
```

Comment:
It seems to be typo in ``getFormate`` function name

Reply:
```suggestion
export const getFormat = seconds => `${addZero(Math.floor(seconds / 60))}:${addZero(seconds % 60)}`
```